tests/lib/rules/no-wait-for-side-effects.test.ts

```diff
@@ -407,6 +445,13 @@ ruleTester.run(RULE_NAME, rule, {
         })
       `,
 					errors: [{ line: 5, column: 11, messageId: 'noSideEffectsWaitFor' }],
+					output: `
+        import { waitFor } from '${testingFramework}';
+        import { renderHelper } from 'somewhere-else';
+        renderHelper(<App />)
+        await waitFor(() => {
+        })
+      `,
```

Comment:
question: Shouldn't this one remove the empty `waitFor`?

Similar to the same case but with an arrow function (the case above).